### PR TITLE
Remove progress bar

### PR DESCRIPTION
**Pull request by svillafe**
# Delete the progress bar from flow B

## Why?
Surprisingly after we ran the first A/B Test, the old design performed better than the new design. Therefore we decide to remove the progress bar from the new design, since our hypothesis is that it distracting the final user.

## Related PRs
https://github.com/guardian/identity-frontend/pull/194

### Desired
new-fow-bw 1

### Final result

screencapture

### CODE REVIEW

**Comment by SiAdcock** [comment]
One minor colour name point, otherwise 👍

File: frontend/assets/stylesheets/components/_payment-b.scss [hunk 1]
```diff
@@ -10,17 +10,22 @@
     }
 
     .form-header {
-    	margin-top: $gs-baseline/2;
+        border-bottom: 1px solid #4bc6df;
```
Comment (svillafe) [inline frontend/assets/stylesheets/components/_payment-b.scss]: Updated 😅

**Comment by svillafe** [comment]
Thanks @SiAdcock and @Ap0c some gems for you 💎 💎 💎 💎 . Spend them wisely.

**Comment by svillafe** [comment]
The changes are working in **PROD**. We are not able to see them because the A/B Test is still off.

**Pull request merged**